content/en/agent/docker/_index.md

```diff
@@ -89,6 +89,7 @@ The Agent's [main configuration file][8] is `datadog.yaml`. For the Docker Agent
 |--------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `DD_API_KEY`       | Your Datadog API key (**required**)                                                                                                                                                                                                                                                                                                              |
 | `DD_HOSTNAME`      | Hostname to use for metrics (if autodetection fails)                                                                                                                                                                                                                                                                                             |
+| `DD_ENV`          | Sets the global `env` tag for all data emitted.                                                                                                                                                                                                                                                                 |
```

Comment:
Let's move it above DD_HOSTNAME (since it is more important, useful, and also prior by alphabetical order)